src/man/sssd.conf.5.xml

```diff
@@ -733,26 +733,24 @@
                             When SSSD switches to offline mode the amount of
                             time before it tries to go back online will
                             increase based upon the time spent disconnected.
-                            This value is in seconds and calculated by the
-                            following:
-                        </para>
-                        <para>
-                             offline_timeout + random_offset
-                        </para>
-                        <para>
-                            The random offset value is from 0 to 30.
+                            By default SSSD uses incremental behaviour to
+                            calculate delay in between retries.
+                            The longer backend is offline, the bigger delays
+                            in between attempts to checking it status again.
                             After each unsuccessful attempt to go online,
                             the new interval is recalculated by the following:
                         </para>
                         <para>
-                            new_interval = (old_interval * 2) + random_offset
+                             delay = (delay * 2) % offline_timeout_max + (rand() % refresh_max_random_offset)
+                        </para>
+                        <para>
+                            The refresh_max_random_offset default value is 30.
+                            The offline_timeout_max default value is 3600.
+                            The end result is amount of seconds before next retry.
```

Comment:
I would mention the default for `offline_timeout` here as well, because the random offset should not only be chosen based on the maximal delay, but the initial delay should be kept in minds as well because the offset is added here as well.

Reply:
Agree, updated.